doc/man3/OSSL_CMP_X509_digest.pod

```diff
@@ -0,0 +1,36 @@
+=pod
+
+=head1 NAME
+
+OSSL_CMP_X509_digest
+- CMP certificate utility functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/cmp_util.h>
+
+ ASN1_OCTET_STRING *OSSL_CMP_X509_digest(const X509 *cert);
```

Comment:
Does this need to be a public function? If so, why is it CMP specific?

Reply:
Due to the mock server code now residing in `apps/` it must be public.
I've moved it to `crypto/x509/x_all.c`, renaming it to `X509_digest_sig()`.